packages/filesystem/src/browser/file-dialog/file-dialog-model.ts

```diff
@@ -42,6 +43,10 @@ export class FileDialogModel extends FileTreeModel {
         return this._initialLocation;
     }
 
+    set disableFileSelection(isSelectable: boolean) {
```

Comment:
should we add the return type @elaihau ?

Reply:
I got this when the return type is added " `[ts] A 'set' accessor cannot have a return type annotation.`

Reply:
ohh sorry you seem to be right, I didn't know about that rule but now I do :)